frontend/src/components/Friend/Friend.jsx

```diff
@@ -0,0 +1,45 @@
+import "./Friend.css";
+import axios from "axios";
+import { useContext } from "react";
+import { UserContext } from "../App/App";
+
+export default function Friend({ username, profile, setProfile }) {
+    const { user, setErrorMessage } = useContext(UserContext);
+
+    const handleFriend = async () => {
+        const url = profile.friend
+            ? "http://localhost:3000/unfriend"
+            : "http://localhost:3000/friend";
```

Comment:
consider moving endpoint names ("friend", "unfriend') to the constants utils, to avoid magic strings in the components

Reply:
I'm using a env variable for the host portion of the urls now.